packages/devtools/lib/src/ui/plotly.dart

```diff
@@ -0,0 +1,237 @@
+@JS()
+library plotly;
+
+import 'package:js/js.dart';
+
+@JS()
+external void hookupPlotlyClick(String domName, Function jsFunction);
+
```

Comment:
overall this JS interop class definition looks great!

Reply:
Not complete but everything we need for now.

Reply:
much better to just add what we need than to spend a lot of time adding whappers for things we don't need yet and might never need

Reply:
much better to just add what we need than to spend a lot of time adding wrappers for things we don't need yet and might never need